CODE_STYLE.md

```diff
@@ -320,6 +320,10 @@ local function check(age, name)
 end
 ```
 
+The function should return `<boolean>`, `err`.
```

Comment:
Change in the doc should not begin with the prefix "fix"

Reply:
@kingluo any updates?

Reply:
we'd better to add some good or bad example, make it clearer